testing/endtoend/minimal_scenario_e2e_test.go

```diff
@@ -26,3 +26,24 @@ func TestEndToEnd_ScenarioRun_EEOffline(t *testing.T) {
 	runner.config.EvalInterceptor = runner.eeOffline
 	runner.scenarioRunner()
 }
+
+func TestEndToEnd_MultiScenarioRun_ValidatorRESTApi(t *testing.T) {
```

Comment:
I think running so many test cases for just the `REST` Api is going to lengthen our test times considerably. Currently the scenario tests take 3 hours to run, with all these additional tests they maybe be pushed up to 6 hours, which isn't really acceptable imo. I think having `TestEndToEnd_MinimalConfig_ValidatorRESTApi` should be enough, @rkapka what do you think ?

Reply:
I agree

Reply:
I removed all tests except `TestEndToEnd_MinimalConfig_ValidatorRESTApi`